src/components/Table/index.tsx

```diff
@@ -166,8 +167,8 @@ export interface TableBaseOptions<D extends object> {
     disableFilters?: boolean;
     /** Disable groupBy */
     disableGroupBy?: boolean;
-    /** Disable row select */
-    disableRowSelect?: boolean;
+    /** When true, disables row select (default: false). Pass an array of row ids to disable selection for specific rows */
```

Comment:
It might be better to use a new props with a callback to determine whether an item is disabled or not?

isItemDisabled | (item: T) =&gt; boolean | Determines whether a given item is disabled. If an item is disabled, the user can't select it.
-- | -- | --

Reply:
Sure happy to swap it to a callback :)